src/landscape/components/LandscapeListMap.css

```diff
@@ -11,15 +11,15 @@
   color: #fff;
 }
 
-.landscape-list-map-cluster-icon {
+.landscape-list-map-cluster-icon.leaflet-marker-icon.leaflet-interactive {
   background-clip: padding-box;
-  background-color: #98532999;
+  background: #98532999;
   border-radius: 20px;
 }
 
-.landscape-list-map-marker-icon {
+.landscape-list-map-marker-icon.leaflet-marker-icon.leaflet-interactive {
   background-clip: padding-box;
-  background-color: #985329;
+  background: #985329;
```

Comment:
question: did you mean to change this? You're also overriding repeat, position, etc.

Reply:
@paulschreiber yes, this was intentional to override a conflicting style in the Map.css file, should I override something else?

Reply:
what are you trying to override?